velox/type/StringView.h

```diff
@@ -86,11 +93,16 @@ struct StringView {
   //
   /* implicit */ StringView(const char* data)
       : StringView(data, strlen(data)) {}
+
   explicit StringView(const folly::fbstring& value)
       : StringView(value.data(), value.size()) {}
+  explicit StringView(folly::fbstring&& value) = delete;
```

Comment:
I think `folly::fbstring` are already deprecated at this point.. Maybe not on this PR but we could probably remove it.